cli/command/volume/create.go

```diff
@@ -52,7 +52,7 @@ func newCreateCommand(dockerCli command.Cli) *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if len(args) == 1 {
 				if options.name != "" {
-					return errors.Errorf("conflicting options: either specify --name or provide positional arg, not both")
+					return errors.Errorf("conflicting options: cannot specify a volume-name through both --name and as a positional arg")
```

Comment:
is there a specific (i'd guess historical) reason this command has support for both an option and a positional arg for the name?

Reply:
Yeah, there was some back-and-forth whether name should be a positional arg or a flag. The `--name` was changed to a positional arg, but because it already shipped was kept (but soft-deprecated / hidden).

Reply:
I think I may have been partially responsible for that, considering that `docker volume create foobar` is  more convenient than `docker volume create --name=foobar`, but later discussions were that, because name is _optional_, using a `--flag` is a more common convention.